### PR TITLE
emit job status lifecycle in event replayer

### PR DESCRIPTION
**Pull request by chrismeyersfsu**
Help UI debug stdout.

### CODE REVIEW

File: awx/main/management/commands/replay_job_events.py [hunk 1]
```diff
@@ -210,12 +236,15 @@ def add_arguments(self, parser):
...
         parser.add_argument('--skip-range', dest='skip_range', type=str, metavar='k',
                             default='0:-1:1', help='Range of events to skip')
+        parser.add_argument('--random-seed', dest='random_seed', type=int, metavar='r',
+                            default=7, help='Seed to use for random number generation')
 
     def handle(self, *args, **options):
         job_id = options.get('job_id')
         speed = options.get('speed') or 1
         verbosity = options.get('verbosity') or 0
+        random_seed = options.get('random_seed')
```
Comment (jakemcdermott) [inline awx/main/management/commands/replay_job_events.py:255]: 👍

**Review by jakemcdermott — APPROVED** [review]

File: awx/main/management/commands/replay_job_events.py [hunk 2]
```diff
@@ -76,9 +91,10 @@ def get_job_events(self, job):
             job_events = job.inventory_update_events.order_by('created')
         elif type(job) is SystemJob:
             job_events = job.system_job_events.order_by('created')
-        if job_events.count() == 0:
+        count = job_events.count()
+        if count == 0:
             raise RuntimeError("No events for job id {}".format(job.id))
-        return job_events
+        return job_events, count
```
Comment (jakemcdermott) [inline awx/main/management/commands/replay_job_events.py]: this makes sense, a few places in the command unit test might need to be updated to anticipate the additional return vals

File: awx/main/management/commands/replay_job_events.py [hunk 3]
```diff
@@ -76,9 +91,10 @@ def get_job_events(self, job):
...
```
Comment (jakemcdermott) [inline awx/main/management/commands/replay_job_events.py:98]: this makes sense, a few places in the command unit test might need to be updated to anticipate the additional return vals

Comment (chrismeyersfsu) [reply awx/main/management/commands/replay_job_events.py:98]: wow, future me needs to give past me a high five. Forgot I had tests.

**Pull request merged**